src/locale/locales/it/messages.po

```diff
@@ -651,16 +651,16 @@ msgstr ""
 #: src/view/com/profile/FollowButton.tsx:36
 #: src/view/com/profile/FollowButton.tsx:46
 msgid "An issue occurred, please try again."
-msgstr "Si è verificato un problema, riprova un'altra volta."
+msgstr "Si è verificato un problema, per favore riprova più tardi."
 
 #: src/screens/Onboarding/StepInterests/index.tsx:219
 msgid "an unknown error occurred"
-msgstr "si è verificato un errore sconosciuto"
+msgstr "Si è verificato un errore sconosciuto"
 
 #: src/components/moderation/ModerationDetailsDialog.tsx:151
 #: src/components/moderation/ModerationDetailsDialog.tsx:147
 msgid "an unknown labeler"
-msgstr ""
+msgstr "Etichettatore sconosciuto"
```

Comment:
```suggestion
msgstr "un etichettatore sconosciuto"
```
This string is part of a sentence, so I think it should start with a lowercase letter